MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -2037,11 +2042,15 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 		*pDataSize = 0;
 		return reportError(VK_INCOMPLETE, "Error writing pipeline cache data: %s", ex.what());
 	}
+#else
+	MVKAssert(false, "Pipeline cache serialization is unavailable. To enable pipeline cache serialization, build MoltenVK without the MVK_EXCLUDE_CEREAL build setting.");
```

Comment:
Not sure that `MVKAssert` is appropriate here, as it will crash the app in a Debug build. It is probably better to just simply set `*pDataSize = 0;` and report the result as incomplete with an appropriate error message, as you are doing. 

And do make sure you set `*pDataSize = 0;`, to indicate nothing was exported, for those cases where the assert doesn't cause a crash.

Reply:
Ok, done.